include/revng/Model/Argument.h

```diff
@@ -38,13 +39,10 @@ TUPLE-TREE-YAML */
 class model::Argument : public model::generated::Argument {
 public:
   using generated::Argument::Argument;
+  Argument(uint64_t Index, UpcastableType &&Type) :
+    model::generated::Argument(Index, std::move(Type), {}, {}, {}) {}
 
   Identifier name() const;
-
-public:
-  bool verify() const debug_function;
-  bool verify(bool Assert) const debug_function;
-  RecursiveCoroutine<bool> verify(VerifyHelper &VH) const;
```

Comment:
Why?

Reply:
Because it's body was so basic I just inlined it in:
```cpp
  rc_return VH.maybeFail(CustomName().verify(VH)
                         and rc_recur Type().verify(VH));
```

I'm pretty sure everyone who'd consider using this is better of verifying the final function instead.

Reply:
`debug_function` are useful in `gdb` though

Reply:
Done!